configure

```diff
@@ -2739,8 +2739,14 @@ check_build_tools()
 	                  "C compiler" "yes" found_cc
 
 	# Also check the compiler to see if we are (cross-)compiling for Windows
-	if "${found_cc}" -dM -E - < /dev/null 2> /dev/null | grep -q _WIN32; then
```

Comment:
I would suggest using `grep -q "#define _WIN32"` instead of checking `os_name`

Reply:
Thank you for the suggestion. I have updated it following your idea: 618afd3